README.md

```diff
@@ -88,12 +117,13 @@ Many things all at once:
 - Then parsing XML
 - Then doing map-reduce data from Wikipedia
 
-The main example is in [FindMostPopularWikipediaKeywordsFs2App](example/src/main/scala/xs4s/example/FindMostPopularWikipediaKeywordsFs2App.scala).
+The main example is in [FindMostPopularWikipediaKeywordsFs2App](example/src/main/scala/xs4s/example/FindMostPopularWikipediaKeywordsFs2App.scala) or [FindMostPopularWikipediaKeywordsZIOApp](example/src/main/scala/xs4s/example/FindMostPopularWikipediaKeywordsZIOApp.scala).
 There is also a plain Scala example (using `Iterator`) in [FindMostPopularWikipediaKeywordsPlainScalaApp](example/src/main/scala/xs4s/example/FindMostPopularWikipediaKeywordsPlainScalaApp.scala).
 
 ```bash
 $ git clone https://github.com/ScalaWilliam/xs4s.git
 $ sbt "examples/runMain xs4s.example.FindMostPopularWikipediaKeywordsFs2App" 
+$ sbt "examples/runMain xs4s.example.FindMostPopularWikipediaKeywordsZIOApp" 
```

Comment:
Feel free to add yourself to the Authors list here too; I'm very grateful for your contribution.